PsfLauncher/StartProcessHelper.h

```diff
@@ -4,7 +4,7 @@
 #include "Globals.h"
 #include <wil\resource.h>
 
-HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, DWORD timeout, LPPROC_THREAD_ATTRIBUTE_LIST attributeList = nullptr)
+HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, DWORD timeout, LPPROC_THREAD_ATTRIBUTE_LIST attributeList = nullptr, DWORD *exitCode = nullptr)

```

Comment:
a pointer to a DWORD indicates it is both input and output.  I only see it being used on line 54.  Is it used as input anywhere?

Reply:
I am passing a DWORD pointer here to collect the exitCode on launching the powershell process (only to collect the output) and use it to display the error code in PsfPowershellScriptRunner.h

Reply:
If this is only for output please add _Out_ to signify this is an output parameter.